core/src/it/resources/restonomer_context/checkpoints/authentication/jwt_authentication/checkpoint_jwt_authentication.conf

```diff
@@ -0,0 +1,22 @@
+name = "checkpoint_jwt_authentication"
+
+request = {
+  url = "http://localhost:8080/jwt-auth-token"
+
+  authentication = {
+    type = "jwt-authentication"
+    subject = "jwt-authentication"
```

Comment:
choose different subject (not same as type)

Reply:
resolved